saleor/graphql/order/tests/test_fulfillment.py

```diff
@@ -1017,3 +1017,41 @@ def test_fulfillment_query(
         "orderLine": {"id": order_line_2_id},
         "quantity": order_line_2.quantity,
     } in fulfillment_data["lines"]
+
+
+QUERY_ORDER_FULFILL_DATA = """
+query OrderFulfillData($id: ID!) {
+    order(id: $id) {
+        id
+        lines {
+            variant {
+                stocks {
+                    warehouse {
+                        id
+                    }
+                    quantity
+                    quantityAllocated
+                }
+            }
+        }
+    }
+}
+"""
+
+
+def test_staff_can_query_order_fulfill_data(
+    staff_api_client, order, permission_manage_orders
+):
+    order_id = graphene.Node.to_global_id("Order", order.pk)
+    variables = {"id": order_id}
+    response = staff_api_client.post_graphql(
+        QUERY_ORDER_FULFILL_DATA, variables, permissions=[permission_manage_orders]
+    )
+    _ = get_graphql_content(response)
```

Comment:
could we check if the query actually return something instead of testing if it doesn't break?